tests/test_spatial_tools.py

```diff
@@ -13,6 +13,7 @@
 import geoutils as gu
 import numpy as np
 import rasterio as rio
+from osgeo import gdal
```

Comment:
rasterio strongly [discourage](https://rasterio.readthedocs.io/en/latest/topics/switch.html) importing rio and gdal in the same script... This could lead to massive bugs at some point. But I guess it's ok if gdal is only used in the hillshade test...

Reply:
Actually, maybe gdal should only be imported within the make_gdal_hillshade function?

Reply:
Moved it to the function in 4d33081